docs/sdd.md

```diff
@@ -20,9 +20,9 @@ Status | Draft
 
 Document | Description
 --- | ---
-:construction: High-Level Concept of Operations (CONOPS) | Overview of Arrow microservices.
-:construction: High-Level Interface Control Document (ICD) | Interfaces and frameworks common to all Arrow microservices.
-:construction: Requirements - `svc-FIXME` | Requirements and user stories for this microservice.
+[High-Level Concept of Operations (CONOPS)](https://github.com/Arrow-air/se-services/blob/develop/docs/conops.md) | Overview of Arrow microservices.
+[High-Level Interface Control Document (ICD)](https://github.com/Arrow-air/se-services/blob/develop/docs/icd.md)  | Interfaces and frameworks common to all Arrow microservices.
+[Requirements - `svc-FIXME`](https://nocodb.arrowair.com/dashboard/#/nc/view/d1bb0a51-e22f-4b91-b1c5-66f11f4f861b) | Requirements and user stories for this microservice.
```

Comment:
I think there's a way to generate a link for a specific svc. See svc-assets or pricing.

Reply:
Replaced with a FIXME to use a URL for your module's specific view: https://github.com/Arrow-air/svc-template-rust/pull/22/commits/fc42b589509567e5059fb64c73e66fb672e927b1